portafly/src/components/pages/accounts/listing/AccountsListingTable.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect } from 'react'
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  OnSelect,
+  OnSort
+} from '@patternfly/react-table'
+import {
+  PageEmptyState,
+  useDataListFilters,
+  useDataListPagination,
+  useDataListTable,
+  useDataListBulkActions,
+  Toolbar,
+  SearchWidget,
+  PaginationWidget,
+  BulkSelectorWidget,
+  SendEmailModal,
+  ChangeStateModal,
+  BulkActionsWidget,
+  filterRows
+} from 'components'
+import { useTranslation } from 'i18n/useTranslation'
+import { ToolbarContent, ToolbarItem } from '@patternfly/react-core'
+
+const AccountsListingTable: React.FunctionComponent = () => {
+  const { t } = useTranslation('audienceAccountsListing')
+  const {
+    columns,
+    rows,
+    sortBy,
+    setSortBy,
+    selectedRows,
+    selectOne
+  } = useDataListTable()
+
+  if (rows.length === 0) {
```

Comment:
Not mandatory and is still readable,  but the `length` method will always return integer, and `0` is `falsey`, you could force the boolean value by `!!`

Reply:
I don't understand. The point of this predicate is to compare one integer to another.

Reply:
just pointing out that `rows.length` is either `0` => falsey or any integer => `truthy`

Reply:
Ah! gotcha.

Reply:
So, between `!rows.length` and `rows.length === 0`, since there's already a logical operatior involved I would choose the second.